.github/workflows/conda-create-cuda10.1.yml

```
@@ -0,0 +1,34 @@
+name: Test CUDA 10.1 conda env
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '30 14 * * *'
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'     
+        required: true
+        default: 'warning'
+      tags:
+        description: 'Test scenario tags'  
+
+jobs:
+  build-linux:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 5
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    - name: Add conda to system path
+      run: |
+        # $CONDA is an environment variable pointing to the root of the miniconda directory
+        echo $CONDA/bin >> $GITHUB_PATH
+    - name: Create Conda environment with CUDA 10.1
+      run: |
+        conda env create --file conda/environments/nvtabular_dev_cuda10.1.yml --name nvtabular_dev_10.1
```

Thanks for this!

It makes sense to run each cuda environment in its own GA job - but I'm wondering if we can cut down on the boilerplate by using a build matrix for this, rather than have as 3 separate files.

Something like:
```
  build-linux-cross:
    runs-on: ubuntu-latest
    strategy:
      matrix:
        target: [cuda11.0, cuda10.2, cuda10.1]

```
And then changing the run step to use the build matrix

```suggestion
        conda env create --file conda/environments/nvtabular_dev_{{ matrix.target }}.yml --name nvtabular_dev_{{ matrix.target }}
```

---

Yeah, that could work. 👍